android/KMEA/app/src/main/java/com/tavultesoft/kmea/data/Keyboard.java

```diff
@@ -150,7 +150,7 @@ protected void fromJSON(JSONObject installedObj) {
       this.isNewKeyboard = installedObj.getBoolean(KB_NEW_KEYBOARD_KEY);
       this.font = installedObj.getString(KB_FONT_KEY);
       this.oskFont = installedObj.getString(KB_OSK_FONT_KEY);
-      this.displayName = installedObj.getString(KB_DISPLAY_NAME_KEY);
+      this.displayName = installedObj.optString(KB_DISPLAY_NAME_KEY);
```

Comment:
```suggestion
      this.displayName = installedObj.optString(KB_DISPLAY_NAME_KEY, null);
```

This matches `master`?

Reply:
Right.  The default value there is `""`, not `null`, so we do need the second param.